src/redist/targets/Branding.targets

```diff
@@ -6,6 +6,7 @@
       <MSBuildExtensionsBrandName>.NET Standard Support for Visual Studio 2015</MSBuildExtensionsBrandName>
       <SharedFrameworkBrandName>Microsoft .NET Core Runtime $(MicrosoftNETCoreAppPackageVersion)</SharedFrameworkBrandName>
       <NetCoreAppTargetingPackBrandName>Microsoft .NET Core Targeting Pack $(MicrosoftNETCoreAppPackageVersion)</NetCoreAppTargetingPackBrandName>
+      <NetStandardTargetingPackBrandName>Microsoft .NET Standard 2.1 Targeting Pack $(MicrosoftNETCoreAppPackageVersion)</NetStandardTargetingPackBrandName>
```

Comment:
Including 2.1 in the name makes sense to me, since it doesn't align with the 3.0 everywhere else. Are there other reasons for it though?

Reply:
@dagood 
I'm expecting to insert .NET Standard 2.0 Targeting Packs